src/interactors/interactors.test.js

```diff
@@ -0,0 +1,94 @@
+import Cloud, * as cloudMocks from '@cesarbr/knot-fog-connector-knot-cloud';
+import RegisterDevice from './RegisterDevice';
+import UnregisterDevice from './UnregisterDevice';
+import UpdateSchema from './UpdateSchema';
+import PublishData from './PublishData';
+
+jest.mock('@cesarbr/knot-fog-connector-knot-cloud');
+
+const mockThing = {
+  id: 'abcdef1234568790',
+  name: 'my-device',
+  schema: [
+    {
+      sensorId: 0,
+      typeId: 65521,
+      valueType: 3,
+      unit: 0,
+      name: 'bool-sensor',
+    },
+  ],
+};
+const mockData = [
+  {
+    sensorId: 0,
+    value: true,
+  },
+];
+
+describe('Interactors', () => {
+  beforeEach(() => {
+    cloudMocks.mockAddDevice.mockClear();
+    cloudMocks.mockRemoveDevice.mockClear();
+    cloudMocks.mockUpdateSchema.mockClear();
+    cloudMocks.mockPublishData.mockClear();
+  });
+
+  test('registerDevice: should execute interactor correctly when registering a new thing on cloud', async () => {
```

Comment:
What do you think about removing those prefixes (`registerDevice:`, etc)? At least for me, the description is already clear this is a test related to register operation.

Reply:
Done!